Project.toml

```diff
@@ -3,24 +3,24 @@ uuid = "30fc2ffe-d236-52d8-8643-a9d8f7c094a7"
 version = "0.6.0"
 
 [deps]
-InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 LearnBase = "7f8f8fb0-2700-5f03-b4bd-41f8cfc144b6"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-LearnBase = "0.3"
+LearnBase = "0.4"
```

Comment:
```suggestion
LearnBase = "0.3, 0.4"
```

Reply:
Thank you. In this case, we had a minor release in `LearnBase.jl` that is breaking. In particular, `LearnBase.jl` doesn't export all the names as before, and we moved the `AggMode` submodule there (which we will deprecate in the next round of improvements).

I am following this design where developers need to explicitly import specific concepts from `LearnBase.jl` to define methods, and end-user packages such as `LossFunctions.jl` only export a subset of the API to avoid namespace pollution with other ML projects outside the JuliaML umbrella.